### DISCUSSION TITLE
Fatal error while updating OMZ

### DISCUSSION THREAD

**Discussion by getangar** [ohmyzsh/ohmyzsh#11046] [Help]
Hi community,
I'm using oh-my-zsh on my Mac mini M1 running the latest version of macOS (12.4).
Every time there's an update and I'm asked if I want to install it, the following error appears:

```
[oh-my-zsh] Would you like to update? [Y/n] Y
Updating Oh My Zsh
fatal: pack has 58 unresolved deltas
fatal: fetch-pack: invalid index-pack output
There was an error updating. Try again later?
```

Oh-my-zsh has been installed as per documentation and the strange thing is that I installed it also on the MacBook Air M1, following the same procedure, but on the notebook, I can update without any issue (software configuration of the two computer its almost the same).

Do you have any idea how can I solve this annoying problem? Thanks in advance!
<img width="487" alt="Screenshot 2022-07-12 at 22 05 56" src="https://user-images.githubusercontent.com/41167983/178584425-7dce4fdf-edf6-4208-826a-33d47ab6415a.png">

**Comment by 7system7** [comment]
I have the same issue.

**Comment by roeniss** [comment]
Too less information. I'm using mac Mini m4 and have no problem like that. How to reproduce?

**Comment by 7system7** [comment]
I did a backup about the `custom` folder and the `.zshrc`, resintalled the oh-my-zsh, and put back the folder and config. Now, it works.

**Comment by roeniss** [comment]
it might be the problem of git version. AFAIR, default installed git version was too low. I'm using `git version 2.48.1`. Maybe that's the reason?

**Comment by 7system7** [comment]
Nope. I use Arch btw. So, the git version is not a problem.

```bash
$ yay -Qi git
Name            : git
Version         : 2.49.0-1
Description     : the fast distributed version control system
Architecture    : x86_64
URL             : https://git-scm.com/
Licenses        : GPL-2.0-only
Groups          : None
Provides        : None
Depends On      : curl  expat  perl  perl-error  perl-mailtools  openssl  pcre2  grep  shadow  zlib
Optional Deps   : ...
Required By     : yay
Optional For    : gettext  npm
Conflicts With  : None
Replaces        : None
Installed Size  : 28,71 MiB
Packager        : Christian Heusel <gromit@archlinux.org>
Build Date      : 2025. márc. 14., péntek, 18:20:20 CET
Install Date    : 2025. márc. 15., szombat, 20:21:13 CET
Install Reason  : Explicitly installed
Install Script  : Yes
Validated By    : Signature
```
